tests/test_Admin.py

```diff
@@ -835,17 +835,17 @@ def test_alter_consumer_group_offsets_api():
         a.alter_consumer_group_offsets([request_with_group_and_topic_partition_offset1,
                                         same_name_request])
 
-    fs = a.alter_consumer_group_offsets([request_with_group_and_topic_partition_offset1])
-    with pytest.raises(KafkaException):
-        for f in fs.values():
-            f.result(timeout=10)
-
-    fs = a.alter_consumer_group_offsets([request_with_group_and_topic_partition_offset1],
-                                        request_timeout=0.5)
-    for f in concurrent.futures.as_completed(iter(fs.values())):
-        e = f.exception(timeout=1)
-        assert isinstance(e, KafkaException)
-        assert e.args[0].code() == KafkaError._TIMED_OUT
+    # with pytest.raises(KafkaException):
```

Comment:
Why the code is commented?

Reply:
This is raising Fatal error intermittently in Python build for MacOS. I am not able to reproduce this. Not able to reproduce the issue even with M2 mac.

Reply:
I will add a PR to enable them again.

Reply:
ok, please add a comment for now